fairscale/optim/oss.py

```diff
@@ -107,6 +107,14 @@ def __init__(
         # (partition and optimizer for the shard)
         self.refresh_trainable()
 
+    def zero_grad(self, set_to_none: Optional[bool] = False) -> None:
+        logging.warning(
```

Comment:
this was already true before this PR, I just figured that would give an easier clue as of what to do if this error pops up